src/lib/slangroom/services.ts

```diff
@@ -1,5 +1,7 @@
-import { PB, slangroom } from '.';
+import { slangroom } from '.';
+import { PUBLIC_BACKEND_URL } from '$env/static/public';
 
+const pb = PUBLIC_BACKEND_URL;
```

Comment:
Awesome!
I do not understand why this pb new constant (assign a constant to another constant it's a rename or aliasing) is needed